content/installation/java/package/scheduled-update-curl.md

```diff
@@ -0,0 +1,131 @@
+<!--
+title: "Schedule Java agent Updates with curl"
+description: "How to use cron and curl to schedule Java agent updates from Maven Central"
+tags: "java agent deployment maven central cron curl auto-update update"
+-->
+
+## About Updates with Maven Central 
+
+Some users like to automatically update their Contrast Java agent software to the latest version. Linux users can schedule Java agent updates from Maven Central using common Linux tools `cron` and `curl`. The following instructions show you how to configure a scheduled Java agent job on an Ubuntu 18.04 Linux host.
+
+If you want to perform each step as you follow along with this guide, you can use [Vagrant](https://www.vagrantup.com/) to create a new Ubuntu 18.04 virtual machine.
+
+```console
+vagrant init ubuntu/bionic64
+```
+
+```console
+vagrant up
+```
+
+```console
+vagrant ssh
+```
+
+## Step 1: Install Contrast
+
+* Create a shared directory for Contrast software:
+
+```console
+sudo mkdir -p /opt/contrast
+```
+
+* Create a script for installing the latest Java agent in the */etc/cron.daily* directory. Scripts in this directory execute once daily; as a result, the host updates the latest Java agent each day.
+
+* Use `tee` to create this script. Press `CTRL+D` when you've finished typing all the lines:
+
+```console
+$ sudo tee -a /etc/cron.daily/install-latest-contrast-agent > /dev/null
+#!/bin/bash -u
+
+CONTRAST_DIRECTORY=/opt/contrast
+CONTRAST_FILE_NAME=contrast-agent.jar
+
+curl --fail --silent --location "https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.contrastsecurity&a=contrast-agent&v=LATEST" -o /tmp/$CONTRAST_FILE_NAME
+if [ $? -ne 0 ]; then
+    echo "Failed to download Contrast Java agent" >&2
+    exit 1
+fi
+mv /tmp/$CONTRAST_FILE_NAME $CONTRAST_DIRECTORY/$CONTRAST_FILE_NAME
+```
+
+* Set the execute bit on the new script file:
+
+```console
+sudo chmod +x /etc/cron.daily/install-latest-contrast-agent
+```
+
+* To test the script, execute it and then verify that the file exists using `stat`:
+
+```console
+$ sudo /etc/cron.daily/install-latest-contrast-agent
+$ stat /opt/contrast/contrast-agent.jar
+stat /opt/contrast/contrast-agent.jar
+  File: /opt/contrast/contrast-agent.jar
+  Size: 10568283        Blocks: 20648      IO Block: 4096   regular file
+Device: 801h/2049d      Inode: 256034      Links: 1
+Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
+Access: 2019-04-11 02:02:01.265775928 +0000
+Modify: 2019-04-11 02:24:47.849796936 +0000
+Change: 2019-04-11 02:24:47.849796936 +0000
+ Birth: -
+```
+
+## Step 2: Gather Contrast account information 
+
```

Comment:
Re-commenting: 

@gilday I made this its own step to mirror the last Maven article.

Reply:
ok